src/base/preferences.cpp

```diff
@@ -448,6 +448,26 @@ void Preferences::setActionOnDblClOnTorrentFn(int act)
     setValue("Preferences/Downloads/DblClOnTorFn", act);
 }
 
+bool Preferences::deleteAddedTorrents() const
```

Comment:
The thing is there is no single function which does the torrent import (and **that** is bad design).  Hence the class to reuse the code. The preferences are in the `TorrentFileProcessignGuard` now.